packages/azure-functions/ProcessMaliciousUploads/index.mjs

```diff
@@ -0,0 +1,89 @@
+import { blobStorageConnector } from '@defra/bng-connectors-lib'
+
+const MALWARE_SCANNNING_RESULT_TAG_NAME = 'Malware Scanning scan result'
+const MALWARE_SCANNNING_RESULT_TAG_VALUE = 'Malicious'
+
+const containerName = 'customer-uploads'
+const tags = `"${MALWARE_SCANNNING_RESULT_TAG_NAME}" = '${MALWARE_SCANNNING_RESULT_TAG_VALUE}'`
+const config = {
+  containerName,
+  tags
+}
+
+let itemProcessingTimeout
+let processingTimeout
```

Comment:
does this need to be at this scope? its only used in 1 function

Reply:
These variables are at module level to facilitate lazy initialisation. The associated environment variables are parsed on first use and the resulting timeouts are stored in these variables.